stdlib/Markdown/test/runtests.jl

```diff
@@ -1293,3 +1293,9 @@ end
     # see issue #42139
     @test md"<一轮红日初升>" |> html == """<p>&lt;一轮红日初升&gt;</p>\n"""
 end
+
+@testset "Docstrings" begin
+    undoc = Docs.undocumented_names(Markdown)
+    @test_broken isempty(undoc)
+    @test undoc == [:html, :latex]
```

Comment:
Once #52743 is merged:
```suggestion
    @test undoc == [Symbol("@doc_str"), Symbol("@md_str"), :html, :latex]
```